README.md

```diff
@@ -188,10 +222,10 @@ compiled with appropriate `target_feature`s, so this cannot occur.
 Benchmarks are run using [`criterion.rs`][criterion]:
 
 ```sh
-cargo bench --features "alloc"
+cargo bench --features "rand_core"
 # Uses avx2 or ifma only if compiled for an appropriate target.
-export RUSTFLAGS="-C target_cpu=native"
-cargo +nightly bench --features "alloc simd_backend"
+export RUSTFLAGS='--cfg curve25519_dalek_backend="simd" -C target_cpu=native'
```

Comment:
Why is `-C target_cpu=native` in here? I remember looking into what this actually does, and it sets avx2 but it doesn't set avx512ifma even on a machine with that feature. What should the recommendation be?

Reply:
Previous benchmarking showed it provided the best optimizations at the cost of non-relocatability

Reply:
Hmm ok. Maybe we should recommend that as a "I don't know what to do" option for the simd backend. I can do that tn or tomorrow.